### PR TITLE
Fixed some file paths raising warnings

### PR DESCRIPTION
**Pull request by HugoFara**
Everything is said in the title, for one file PHP code was forgotten and the other I explicitely stated to go to the index file.

It was raising colourful warnings in my terminal, which is distracting, but no big deal here.

### CODE REVIEW

File: templates/base.html.twig [hunk 1]
```diff
@@ -6,9 +6,9 @@
     <link rel="shortcut icon" href="/favicon.ico" type="image/x-icon"/>
 
     <meta name="viewport" content="width=900" />
-    <link rel="apple-touch-icon" href="/img/apple-touch-icon-57x57.png');?>" />
```
Comment (jzohrab) [inline templates/base.html.twig:9]: Thank you!  Terrible, I missed these when copying files over from LWT.  :-(

File: templates/index.html.twig [hunk 2]
```diff
@@ -51,7 +51,7 @@
 <div id="userlinks" style="margin-top: 30px;">
   <ul>
 
-    <li><a href="/language">Languages</a></li>
+    <li><a href="/language/index">Languages</a></li>
```
Comment (jzohrab) [inline templates/index.html.twig:54]: Hm, when I go to `http://localhost:8080/language/index`, I get the following: `No route found for "GET http://localhost:8080/language/index": Method Not Allowed (Allow: POST)`.

`/language/index` doesn't actually exist ... `/language` is a route mapping in `src/Controller/LanguageController.php`:

```
    #[Route('/', name: 'app_language_index', methods: ['GET'])]
    public function index(LanguageRepository $languageRepository): Response
```

Does that change actually work on your machine?

Comment (HugoFara) [reply templates/index.html.twig:54]: It's does work on my machine (Ubuntu 22). I just used the same pattern as the rest of the code (e. g. ``href="/text/index"`` two lines below). I need to learn more symfony magic I guess :thinking:

Comment (jzohrab) [reply templates/index.html.twig:54]: Interesting, maybe there's an Apache rewrite happening or something.  Symfony's front controller should have throw an exception (that's where it uses the routes).  But if the line `<li><a href="/language">Languages</a></li>` is raising a warning for you, then you could leave this change in, and change the route in the src/Controller/LanguageController.cs.

**Comment by jzohrab** [comment]
@HugoFara - thanks for catching the error. 👍   I've pulled in your changes for the base.html images into the `develop` branch.

I've not pulled in the Language index suggested change b/c it's working for me -- but if it's raising warnings for you, let me know and I'll fix the routes.

**Pull request closed without merging**